model/Human-GEM.yml

```diff
@@ -81022,11 +81002,11 @@
           - MAM02899m: 1
       - lower_bound: 0
       - upper_bound: 1000
-      - gene_reaction_rule: "ENSG00000105953 or ENSG00000181192 or ENSG00000197444"
+      - gene_reaction_rule: "(ENSG00000105953 or ENSG00000181192 or ENSG00000197444) and ENSG00000119689"
```

Comment:
it is not a standard gpr rule.  (ENSG00000105953 and ENSG00000119689)  or (ENSG00000181192 and ENSG00000119689) or (ENSG00000197444 and ENSG00000119689)